doc/man3/ASYNC_start_job.pod

```diff
@@ -25,6 +26,11 @@ ASYNC_get_current_job, ASYNC_block_pause, ASYNC_unblock_pause, ASYNC_is_capable
 
  int ASYNC_is_capable(void);
 
+ int ASYNC_set_mem_functions(ASYNC_stack_alloc_fn alloc_fn,
```

Comment:
include the typedefs for `ASYNC_stack_alloc_fn` and `ASYNC_stack_free_fn` in the synopsis